### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by pamparam83**
fixed errors and updated the code a little

### CODE REVIEW

**Review by samdark — APPROVED** [review]

File: blog/src/User/Console/CreateCommand.php [hunk 1]
```diff
@@ -41,25 +46,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+                'passwordVerify' => $password,
+            ], scope: '');
             $user = $this->signupForm->signup();
         } catch (Throwable $t) {
             $io->error($t->getMessage() . ' ' . $t->getFile() . ' ' . $t->getLine());
-
             return $t->getCode() ?: ExitCode::UNSPECIFIED_ERROR;
         }
 
         if ($user === false) {
```
Comment (rossaddison) [inline blog/src/User/Console/CreateCommand.php:61]: Psalm Level 1: Testing
ERROR: TypeDoesNotContainType - src/User/Console/CreateCommand.php:78:13 - Type App\User\User for $user is never falsy (see https://psalm.dev/056)
        if (null==$user) {

Comment (rossaddison) [reply blog/src/User/Console/CreateCommand.php:61]: Recommended: if (!$user instanceof User) {

File: blog/src/User/Console/CreateCommand.php [hunk 2]
```diff
@@ -12,14 +12,19 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Throwable;
+use Yiisoft\FormModel\FormHydrator;
```
Comment (rossaddison) [inline blog/src/User/Console/CreateCommand.php:15]: Psalm Level 1 Testing Recommendation: Add the following to support recommendation below: 

use App\User\User;

File: blog/src/User/Console/CreateCommand.php [hunk 3]
```diff
@@ -41,25 +46,21 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
             $io->error($t->getMessage() . ' ' . $t->getFile() . ' ' . $t->getLine());
-
             return $t->getCode() ?: ExitCode::UNSPECIFIED_ERROR;
         }
 
         if ($user === false) {
-            $errors = $this->signupForm->getFormErrors()->getFirstErrors();
-            array_walk($errors, fn ($error, $attribute) => $io->error("$attribute: $error"));
-
+            $errors = $this->signupForm->getValidationResult()?->getErrorMessagesIndexedByAttribute();
```
Comment (rossaddison) [inline blog/src/User/Console/CreateCommand.php:62]: Psalm Level 1 Testing: 
ERROR: TypeDoesNotContainNull - src/User/Console/CreateCommand.php:79:23 - Yiisoft\Validator\Result does not contain null (see https://psalm.dev/090)
            $errors = $this->signupForm->getValidationResult()?->getErrorMessagesIndexedByAttribute();

Comment (rossaddison) [reply blog/src/User/Console/CreateCommand.php:62]: Recommended: Remove the question mark in getValidationResult?

File: blog/src/Auth/Form/SignupForm.php [hunk 4]
```diff
@@ -21,8 +21,8 @@ final class SignupForm extends FormModel implements RulesProviderInterface
     private string $passwordVerify = '';
```
Comment (rossaddison) [inline blog/src/Auth/Form/SignupForm.php:21]: Recommendation: Remove $passwordVerify string. Not used in context.

**Pull request merged**